tests/tests/util/testWithFrontier.ts

```diff
@@ -0,0 +1,123 @@
+import Web3 from "web3";
+
+import { spawn, ChildProcess } from "child_process";
+import { DISPLAY_LOG, PORT, RPC_PORT, SPAWNING_TIME, WS_PORT } from "../constants";
+import { customRequest } from "./web3Requests";
+
+const SPECS_PATH = `./moonbeam-test-specs`;
+
+const FRONTIER_LOG = process.env.FRONTIER_LOG || "info";
+
+const BINARY_PATH = `../../frontier/target/debug/frontier-template-node`;
```

Comment:
Do we want to keep the debug build? Is now the time to finally switch to release both here and in paritytech repo?

I vote yes.

Reply:
sg
https://github.com/PureStake/moonbeam/issues/345